lib/gitscrub.js

```diff
@@ -85,15 +85,21 @@ gitscrub.grabAllReadmes = function(logValue) {
 gitscrub.parseReadMe = function(readme, options, callback) {
   var formattedReadMe = {}
   var header = ""
+  var jotNoteArray = []
   var readMeParsed = Lazy(readme).split("\n")
   readMeParsed.each(function(line) {
-    if (line.substring(0, 2) == "# ") {
+    if (line.substring(0, 2) === "# ") {
       formattedReadMe['title'] = line.substring(2)
-    } else if (line.substring(0, 3) == "## ") {
+    } else if (line.substring(0, 3) === "## ") {
       header = line.substring(3)
-      formattedReadMe[header] = ""
+      jotNoteArray = []
+      formattedReadMe[header] = {text: "", jotNotes: []}
```

Comment:
Instead of using the header itself as a key in your JSON object, would it maybe be better have it stored as a value with the key 'header'? You could instead make an array and push sections organized by header. I'm just thinking from a user perspective they probably want to be able to access the header as the value (eg. var header = formattedReadMe[i].header).